packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooserBody/ColumnChooserBody.component.test.js

```diff
@@ -28,17 +28,48 @@ describe('ColumnChooserBody', () => {
 			t: getDefaultT(),
 		};
 		// When
-		const { container } = render(
+		render(
 			<ColumnChooserProvider value={contextValues}>
 				<Component />
 			</ColumnChooserProvider>,
 		);
-		// Then
+		// Then select all is available and checked
+		const selectAll = screen.getByLabelText(/select all/i);
+		expect(selectAll).toBeChecked();
+
+		// Then items are listed
+		expect(
+			screen.queryByRole('checkbox', {
+				name: 'col1',
+			}),
+		).not.toBeInTheDocument();
+		expect(
+			screen.queryByRole('checkbox', {
+				name: 'col2',
+			}),
+		).not.toBeInTheDocument();
+		expect(document.querySelectorAll('use[xlink:href="#locker-closed:M"]')).toHaveLength(2);
+
+		expect(
+			screen.getByRole('checkbox', {
+				name: 'col3',
+			}),
+		).toBeChecked();
+		expect(
+			screen.getByRole('checkbox', {
+				name: 'col4',
+			}),
+		).not.toBeChecked();
+		expect(
+			screen.getByRole('checkbox', {
+				name: 'col5',
+			}),
+		).toBeChecked();
 		expect(
-			// eslint-disable-next-line testing-library/no-container
-			container.querySelectorAll('.tc-column-chooser-row.theme-tc-column-chooser-row'),
-		).toHaveLength(columns.length + 1);
-		expect(container.firstChild).toMatchSnapshot();
```

Comment:
could you please keep one snapshot per component ?

Reply:
Sure 
Should i remove the rest of the test then ? Not sure it's necessary if we have a snapshot